rosserial_client/cmake/rosserial_client-extras.cmake

```diff
@@ -28,7 +28,6 @@ function(rosserial_generate_ros_lib)
     COMMAND ${CATKIN_ENV} rosrun ${make_libraries_PACKAGE} ${make_libraries_SCRIPT} ${PROJECT_BINARY_DIR}
   )
   add_custom_target(${PROJECT_NAME}_ros_lib DEPENDS ${PROJECT_BINARY_DIR}/ros_lib)
-  add_dependencies(${PROJECT_NAME}_ros_lib rosserial_msgs_genpy std_msgs_genpy)
```

Comment:
The message generation target name have changed from `<pkg_name>_genpy` to `<pkg_name>_generate_messages_py` so they no longer work.

I just realized that the `add_custom_target` does not inherent dependencies. Maybe this should depend directly on `rosserial_msgs` and `std_msgs`?